functests/common.sh

```diff
@@ -77,7 +77,7 @@ function _wait_directcsi_to_start() {
     while [[ $running_count -lt $required_count ]]; do
         echo "$ME: waiting for $(( required_count - running_count )) direct-csi pods to come up"
         sleep $(( required_count - running_count ))
-        running_count=$(kubectl get pods --field-selector=status.phase=Running --no-headers --namespace=direct-csi-min-io | wc -l)
+        running_count=$(kubectl get pods --field-selector=status.phase=Running --no-headers --namespace=directpv-min-io | wc -l)
```

Comment:
The namespace changes are not yet done. We still use `direct-csi-min-io` ns in source. Can we take care of the ns change later? It has to be done carefully as we use certs signed with the name. And also the upgrades has to be tested with these changes.

Reply:
Yeah, this shouldn't be changed